depends/packages/xproto.mk

```diff
@@ -5,7 +5,7 @@ $(package)_file_name=$(package)-$($(package)_version).tar.bz2
 $(package)_sha256_hash=636162c1759805a5a0114a369dffdeccb8af8c859ef6e1445f26a4e6e046514f
 
 define $(package)_set_vars
-$(package)_config_opts=--disable-shared
+$(package)_config_opts=--without-fop --without-xmlto --without-xsltproc
```

Comment:
Nit: Looking at https://gist.github.com/dongcarl/892d7deda8da3ff72317009ecbb07ffb#file-xproto-configure-help-L80-L90, I'm not sure what the specs and prototypes are... If anyone else knows please say so, otherwise we can leave it.

Reply:
From the looks of this [mailing list post](https://lists.x.org/archives/xorg-announce/2010-February/001249.html) they are probably related to documentation generation?

> Document building enablement macros. This allows a builder to skip
document generation if such documents are not desired. They should not
be used to workaround tooling issues, use macros above. By default,
building is enabled. The default value can be changed from configure.ac
by invoking the macro like XORG_ENABLE_SPECS(no)


> XORG_ENABLE_DOCS            ENABLE_DOCS        General documentation
> XORG_ENABLE_DEVEL_DOCS      ENABLE_DEVEL_DOCS  More specifically developer's documentation
> XORG_ENABLE_SPECS           ENABLE_SPECS       More specifically functional specifications

Reply:
Let's just do `--disable-specs`, you can put a comment there for the other flags or don't, either way is fine :-)

Reply:
Updated with `--disable-specs`.